### PR TITLE
Integrate profile

### PR DESCRIPTION
**Pull request by huwd**
### What this PR does
- Integrates Elements from profiles
- Big push on linting and flow typing

### What this PR does not do
- These elements are likely not ready for showtime and need further testing / rationalization: Suggestion is that all new react components are developed out of this repo as part of a npm linked dev setup, and that existing ones are brought into the relevant `atom.js`/`molecule.js` ect and swapped over in profile at a point where we can do QA testing.
- Static List flow remains a problem, these have been flagged as TODO as they really should be fixed, however the repo is now clean enough to develop new profile elements which are on this sprint. Will create a separate issue to clean this up!
- There's still a lot of cruft here, some elements need cleaning up, specifically there are far too many dissimilar elements with similar names like `card`
- We need to DRY out the `types.js` file

### CODE REVIEW

**Comment by huwd** [comment]
A parallel effort will need to be made to rationalize some of the styled elements into the common theme file.

**Comment by huwd** [comment]
Subject to review: @Gisleburt / @thethomaseffect I may change all the TODOs in here to Fixme and open separate issues, if you don't think they're a barrier to this PR being merged?
They should get fixed, but this PR now gives us something we can develop on further for Profiles.
We just need to remember to NOT bump the version in static-lists until we can do further work on those components

File: src/Component/Organism/ProfileCard/ProfileCard.jsx [hunk 1]
```diff
@@ -49,14 +51,14 @@ const CardSection = styled.div`
 // @TODO this should be moved to proper CtaButton and Dry out against ProfileCard
 const ViewProfileButton = styled.a`
   display: flex;
-  justify-content: center;
+  justify-content: center;MemberBase
```
Comment (thethomaseffect) [inline src/Component/Organism/ProfileCard/ProfileCard.jsx]: ?

Comment (huwd) [reply src/Component/Organism/ProfileCard/ProfileCard.jsx]: oh arse, good spot, that's my stupid middle mouse button which is inexplicably tied to Ctrl+V, need to find a way to deactivate that - will amend

Comment (huwd) [reply src/Component/Organism/ProfileCard/ProfileCard.jsx]: Sorted!

Comment (thethomaseffect) [reply src/Component/Organism/ProfileCard/ProfileCard.jsx]: You never pushed your fix and there's two more of these

**Pull request merged**